package.json

```diff
@@ -16,8 +16,8 @@
   },
   "dependencies": {
     "@cmfcmf/docusaurus-search-local": "^1.1.0",
-    "@docusaurus/core": "2.4.1",
-    "@docusaurus/preset-classic": "2.4.1",
+    "@docusaurus/core": "^2.4.1",
+    "@docusaurus/preset-classic": "^2.4.1",
```

Comment:
look at the changes from previous version of the code, don't use `^` , we are sticking to a specific version of docusarus